spec/factory_girl/registry_spec.rb

```diff
@@ -57,4 +57,19 @@
     subject.clear
     expect(subject.count).to be_zero
   end
+
+  it "removes a factory" do
+    subject.register(:object_name, registered_object)
+    expect do
+      expect(subject.remove(:object_name)).to eq(registered_object)
```

Comment:
Hmm, I guess so. That was sort of the natural outcome of the built-in Hash#delete implementation. But I'm not sure if it's neccessary or not, or what it really should be. Do you have thoughts?  Maybe just true/false based on if it was deleted or a no-op? Or always nil? Good review, thanks.